tests/junit_common.py

```diff
@@ -97,6 +97,14 @@ def get_output_path(self, test: str, output_file: str) -> Path:
         """
         raise NotImplementedError
 
+    def get_cgns_tol(self) -> float:
+        """retrieve CGNS test tolerance.
+
+        Returns:
+            tolerance (float): Test tolerance
+        """
+        return self.cgns_tol if hasattr(self, 'cgns_tol') else 1.0e-12
```

Comment:
This pattern is a code smell -- we should either add the attribute as a property to the base class or make this function return a constant and override this function in the Ratel subclass to return the attribute. 

I am good with either, whatever you think is more reasonable.

Reply:
Using a property is probably the more pythonic approach